_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,87 @@
+{% if page.name == "tinymcespellchecker.md" %}
+### `content_langs`
+{% else %}
+## `content_langs`
+{% endif %}
+
+{{ site.requires_5_9v }}
+
+This option sets the languages displayed in the `language` dropdown toolbar button and the `language` menu item. The option accepts an array of language objects. Each language object is a JavaScript object defined using the following properties.
+
+| Name | Type | Requirement | Description |
+| ---- | ---- | ----------- | ----------- |
+| `title` | String | required | The name shown for this language in the UI. |
+| `code` | String | required | The language tag, as defined in [Tags for Identifying Languages (BCP47)](https://www.ietf.org/rfc/bcp/bcp47.txt), for this language. This tag is placed in the `lang` attribute for elements formatted using this language. |
```

Comment:
`tag` could be confusing here, as tag often refers to the element name/type. Can we please come up with another term here please?

Note: I know this is the phrase used in the RFC, but most customers aren't going to be proficient in RFC specs, so it needs to be dumbed down a little. At the very least we should do what [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) does and use "language tag" in quotes and never use "tag" standalone.